src/mixins/legal-api-mixin.ts

```diff
@@ -48,19 +48,49 @@ export default class LegalApiMixin extends Vue {
       })
   }
 
+  /**
+   * Fetches a draft dissolution filing.
+   * @returns a promise to return the draft filing, or null if not found
+   */
+  async fetchDraftDissolution (): Promise<any> {
+    // get the draft filing from the tasks endpoint
+    const url = `businesses/${this.getBusinessId}/tasks`
+    return axios.get(url)
+      .then(response => {
+        const filing = response?.data?.tasks?.find(task => task.filing.hasOwnProperty(FilingTypes.DISSOLUTION))?.filing
+        const filingName = filing?.header?.name
+        const filingId = +filing?.header?.filingId || 0
+
+        if (!filing || filingName !== FilingTypes.DISSOLUTION || !filingId) {
+          throw new Error('Invalid API response')
+        }
+
+        // save Filing ID from the header
+        this.setFilingId(filingId)
+        return filing
+      })
+      .catch((error) => {
+        if (error?.response?.status === NOT_FOUND) {
+          return null // Dissolution not found
```

Comment:
Add a comment why you need this (ie, a dissolution may or may not exist and you need to handle both cases).

Reply:
I don't think it's needed. I followed the pattern of the Temp REG request, but this error is handled if any of the operations failed inside the `.then()` and bubbles up to the try/catch in App.vue
Removed 👍

Reply:
OK. As you mentioned this morning, this UI always starts with a draft Disso.

Reply:
This code is now gone.